tree-sitter-python/TreeSitter/Python/AST.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE DataKinds, DeriveAnyClass, DeriveGeneric, DeriveTraversable, DuplicateRecordFields, TemplateHaskell, TypeApplications #-}
-module TreeSitter.Python.AST where
+module TreeSitter.Python.AST
+( module TreeSitter.Python.AST
+) where
```

Comment:
Explicit export lists are a good idea, but since we’re generating all of these with TH it’s easiest just to export everything declared in the module. (This is equivalent to omitting the explicit export list altogether.)